server/src/main/java/com/linecorp/centraldogma/server/CentralDogma.java

```diff
@@ -521,9 +527,67 @@ private Server startServer(ProjectManager pm, CommandExecutor executor,
 
         sb.service("/title", webAppTitleFile(cfg.webAppTitle(), SystemInfo.hostname()).asService());
 
-        sb.service(HEALTH_CHECK_PATH, HealthCheckService.of());
+        final RequestLogConfig requestLogConfig = cfg.requestLogConfig();
+        boolean logHealthCheck = false;
+        boolean logMetrics = false;
+        if (requestLogConfig != null) {
+            final Function<? super HttpService, LoggingService> loggingService =
+                    LoggingService.builder()
+                                  .logger(requestLogConfig.loggerName())
+                                  .requestLogLevel(requestLogConfig.requestLogLevel())
+                                  .successfulResponseLogLevel(requestLogConfig.successfulResponseLogLevel())
+                                  .failureResponseLogLevel(requestLogConfig.failureResponseLogLevel())
+                                  .successSamplingRate(requestLogConfig.successSamplingRate())
+                                  .failureSamplingRate(requestLogConfig.failureSamplingRate())
+                                  .newDecorator();
+            final Set<RequestLogGroup> requestLogGroups = requestLogConfig.targetGroups();
+            if (requestLogGroups.contains(RequestLogGroup.ALL)) {
+                sb.decorator(loggingService);
+                logHealthCheck = true;
+                logMetrics = true;
+            } else {
+                for (RequestLogGroup logGroup : requestLogGroups) {
```

Comment:
Question: Is it possible that users may want to configure different logging parameters for different paths?

e.g.
```
- METRIC: debug
- API: info
- ...
```

Reply:
Oh... It is so nice! 👍

Reply:
We can implement it by adding different `LoggingService`s.